### ISSUE TITLE
Aliases (`@hourly`, `@weekly`, etc.) don't work with `with_seconds_required` option

### ISSUE THREAD

**Issue by azasypkin** [Hexagon/croner-rust#9]
## Summary

It looks like it's not possible to use the same `Cron` config to accept both aliases like `@hourly` or `@weekly` and standard cron expressions while requiring a seconds specifier.
```rust
let get_cron = |pattern: &str| {
    Cron::new(pattern)
        .with_seconds_required()
        .with_dom_and_dow()
        .parse()
};

get_cron("0 0 * * * *")?; -> works as expected

get_cron("@hourly")?; -> fails with `Pattern must consist of six fields, seconds can not be omitted.`
```

I assume it's because of this mapping: https://github.com/Hexagon/croner-rust/blob/830fc53eb3c06ba22e6de13a4da3234382624e01/src/pattern.rs#L175-L182.

To support both, consumers of `croner` essentially have to re-implement the `handle_nicknames` logic to detect aliases and omit `with_seconds_required`, or even manually convert to standard syntax if the expression is just passed along to be parsed elsewhere (e.g., when `croner` is a transitive dependency). I ran into this issue with https://github.com/mvniekerk/tokio-cron-scheduler, which recently switched to `croner` from `cron`.

Thanks for the great library!

**Comment by Hexagon** [comment]
Thanks, great bug report! 

Are you interested in making a pull request or should i have a look at it?

**Comment by Hexagon** [comment]
Hopefully fixed in `2.0.6`

**Comment by azasypkin** [comment]
Wow, that was fast, thanks a lot @Hexagon!
